exercises/F-strings-methods/exercise2.js

```diff
@@ -1,3 +1,9 @@
 const name = " Daniel  ";
 
-console.log(message);
+let firstName = "Ahmad";
+
+let nameLength = firstName.length;
+
+let message = "My name is " + firstName + " and my name is " + nameLength + " characters long";
+
+console.log(message.trim());
```

Comment:
After you added some extra spaces to your name, you can use the trim() on the variable itself, this way it will remove the extra spaces where it should